index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+      <nav id="navbar">
+        <ul>
+      <header>Bicycle Owner's Manual</header><br>
+      <a class="nav-link" href="#Bike_fit"><li>"Bike fit"</li></a><br>
+      <a class="nav-link" href="#Safety_First"><li>"Safety First"</li></a><br>
+      <a class="nav-link" href="#Mechanical_Safety"><li>"Mechanical Safety"</li></a><br>
+      <a class="nav-link" href="#First_Ride"><li>"First Ride"</li></a><br>
+      <a class="nav-link" href="#Basics"><li>"Basics"</li></a>
+    </ul>
+      
+    </nav>
+     
+    <main id="main-doc">
+      <section class="main-section"><br>
+        <header id="Bike_fit">Bike fit</header> <br>
+        <p>Is your bike the right size? To check, see Section 3.A. If your bicycle is too
+          large or too small for you, you may lose control and fall. If your new bike is not
+          the right size, ask your dealer to exchange it before you ride it.</p>
+          <p>Is the saddle at the right height? To check, see Section 3.B. If you adjust
+            your saddle height, follow the Minimum Insertion instructions in Section 3.B.</p>
+            <code>for (let i = 0; i < cars.length; i++) {
+                text += cars[i] + "<br>";
+              } </code>
+              <ul>
+                <li>Is the saddle at the right height? To check, see Section 3.B. If you adjust
+                  your saddle height, follow the Minimum Insertion instructions in Section 3.B</li>
+                <li>Are saddle and seat post securely clamped? A correctly tightened saddle will
+                  allow no saddle movement in any direction. See Section 3.B</li>
+              </ul>
+      </section><br>
+      <section class="main-section">
+        <header id="Safety_First">Safety First</header><br>
+        <p>Always wear an approved helmet when riding your bike, and follow the
+          helmet manufacturers instructions for fit, use and care</p>
+          <p>Do you have all the other required and recommended safety equipment?
+            See Section 2. Its your responsibility to familiarize yourself with the laws of the
+            areas where you ride, and to comply with all applicable laws</p>
+            <code>for (let i = 0; i < cars.length; i++) {
```

Comment:
Since you have lots of <code> tags and they are all equal, you should consider adding a .class to all of this elements in order to adjust the style for all of them at once.